docs/dialect.md

```diff
@@ -60,7 +61,58 @@ return-type | Y | Indicates the return type of the function. For a list of allow
 
 - __date-function element__  
 The <span style="font-family: courier new">date-function</span> element is a specialized variant of <span style="font-family: courier new">function</span>. In addition to the base formula, you can specify one or more datepart formulas, which are used instead of the generic formula when available.  
-The function <span style="font-family: courier new">name</span> must be one of these: DATEADD, DATEDIFF, DATEFORMAT, DATENAME, DATEPARSE, DATEPART, DATETRUNC.  
+The function <span style="font-family: courier new">name</span> must be one of these: DATEADD, DATEDIFF, DATEFORMAT, DATENAME, DATEPARSE, DATEPART, DATETRUNC.   
+
+  -  __DATEPART without custom start of week__
+    ```xml
+        ...
+        <date-function name='DATEPART' return-type='int'>
+          <formula>CAST(TRUNC(EXTRACT(%1 FROM %2)) AS INTEGER)</formula>
+          <formula part='weekday'>(1 + CAST(EXTRACT(DOW FROM %2) AS INTEGER))</formula>
+          <formula part='week'>CAST(FLOOR((7 + EXTRACT(DOY FROM %2) - 1 + EXTRACT(DOW FROM DATE_TRUNC(&apos;YEAR&apos;, %2))) / 7) AS INTEGER)</formula>
+          <argument type='localstr' />
+          <argument type='datetime' />
+        </date-function>
+        ...
+    ```
+
+    Here the first argument (%1) is grabbed from the `<date-part-group>` in the dialect file. It is used with generic (non-part-specific) date function formulas.
```

Comment:
Remove this part: `It is used with generic (non-part-specific) date function formulas.`

Its actually available in the part-specific ones as well, just in practice we never use %1 in those.